### ISSUE TITLE
The "class="js-Issue-markdown wc-Markdown"" info is displayed after applying a label

### ISSUE THREAD

**Issue by softvision-sergiulogigan** [webcompat/webcompat.com#1829]
After logging an issue and setting the labels from the Webcompat UI, the class="js-Issue-markdown wc-Markdown" info is displayed.
Reproducible both on Staging and Production.

![image](https://user-images.githubusercontent.com/16573771/30694674-b3c40a90-9edd-11e7-8ace-a8476ba429f8.png)

**Comment by karlcow** [comment]
Yup it's happening to me all the time.

**Comment by zoepage** [comment]
Just to clarify and documentation purpose, it should be just hidden by CSS or should something other happen?

**Comment by magsout** [comment]
@zoepage it seems that It misses the element html p, which I believe is added by the JS:

![capture d ecran 2017-09-21 a 21 35 53](https://user-images.githubusercontent.com/1997108/30714993-22c23520-9f15-11e7-8263-43d3836168ec.png)

**Comment by softvision-sergiulogigan** [comment]
> it should be just hidden by CSS

I think yes, it should be hidden.

**Comment by miketaylr** [comment]
@softvision-sergiulogigan does this still happen on staging?

**Comment by softvision-sergiulogigan** [comment]
I'm not seeing it anymore, @miketaylr , but let me try again after you finish deploying the new version on staging.

**Comment by magsout** [comment]
@softvision-sergiulogigan does this still happen with the new version?

**Comment by magsout** [comment]
thanks @softvision-sergiulogigan
